src/platform-apps/channels/channel-view-container.tsx

```diff
@@ -32,22 +36,44 @@ export class Container extends React.Component<Properties> {
     };
   }
 
+  state = { countNewMessage: 0 };
+
   componentDidMount() {
     const { channelId } = this.props;
-
     if (channelId) {
       this.props.fetchMessages({ channelId });
+      this.fetchInterval = setInterval(() => {
+        this.props.fetchMessages({ channelId });
+      }, 20000);
```

Comment:
sorry. i think maybe i didn't look closely enough at this PR the first time through. let's pull the setInterval out of the component. generally things that manage scheduling like this are best encapsulated in a way that nothing which uses it has to account for the fact that it is on a timer. ie. the thing that does the scheduling is a different thing than the part that is doing the work. the only time you should need to account for the fact that something is 'scheduled' or using a timer is when you are directly testing the scheduler itself.

instead of having an interval here, let's just make this a trigger for a scheduler. so, something like this:
```
this.props.startMessageSync({ channelId });
```

you can see an example of how we might want to do the actual scheduling here: https://github.com/m3m3n70/zero-web/blob/development/src/store/sagas/chat-channel-list.ts#L123